samples/config/ubuntu.conf

```diff
@@ -8,6 +8,9 @@ system_id=1
 video0=http://127.0.0.1:8000/camera-def-uvc.xml
 
 [imgcap]
+width=640
```

Comment:
@lbegani Please also update config.sample with possible values/documentation. Things to cover:
- What fields are
- Possible values
- Units
- e.g. what does format "2" mean.

Reply:
done, kindly review

Reply:
Thanks. Reviewed.